README.md

```diff
@@ -1,2 +1,211 @@
-# testpilot-metrics
-JS library that handles metrics events for Test Pilot experiments
+# `testpilot-metrics`
+
+The `testpilot-metrics` library sends pings to Google Analytics and Mozilla's
+internal metrics pipeline. It is designed for use by Test Pilot experiments.
+
+
+## Installation
+
+`npm install testpilot-metrics`
+
+The only file you need in your build chain is `testpilot-metrics.js`. It has no
+dependencies.
+
+
+## Usage
+
+### Quick start
+
+Here's a simple node-style example:
+
+```js
+
+const Metrics = require('testpilot-metrics');
+
+const { sendEvent } = new Metrics({
+  id: '@my-addon',
+  cid: 'some-non-PII-user-ID',
```

Comment:
It may be nice to add an example of how to get this cid, in the testpilot addon we are currently setting this in a simple-storage store. and sending it along with each metric event.

initialization in `exports.main`
``` js
// set our unique identifier for metrics
if (!store.clientUUID) {
   store.clientUUID = require('sdk/util/uuid').uuid().toString().slice(1, -1);
}
```
`sendEvent` call
``` js
const { sendEvent } = new Metrics({
  id: '@'+manifest.name,
  cid: store.clientUUID
});
```